about.html

```diff
@@ -12,53 +12,65 @@
 </head>
 
 <body>
+  
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
+      <header>
+        <img class="header-item" src="/img/lambda-black.png">
```

Comment:
You can use the header here, I would do <header> then <nav> and put the <a> tags in the nav.

Reply:
I don't know how I forgot to do that, I have done that on all my previous work. Thanks for pointing it out.